release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -369,6 +369,19 @@ This release adds improvements related to the following components and concepts.
 
 Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
 
+[id="ocp-4-15-egressfirewall-updates-status-management"]
+==== Status management updates for egress firewall and AdminPolicyBasedExternalRoutes CR
+
+The following updates have been made to the status management of egress firewall policy:
```

Comment:
`of egress firewall` and AdminPolicyBasedExternalRoutes?
btw. I think it should be AdminPolicyBasedExternalRoute[no -s here]?
also, it may be nice to format EgressFirewall and AdminPolicyBasedExternalRoutes in a similar way?